src/app/home-page/widget/entry-boxes/entry-boxes.component.ts

```diff
@@ -0,0 +1,96 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { EntryType } from 'app/shared/enum/entry-type';
+import { EntryUpdateTime, User, UsersService } from 'app/shared/openapi';
+import { UserQuery } from 'app/shared/user/user.query';
+import { debounceTime, takeUntil } from 'rxjs/operators';
+import { WorkflowService } from 'app/shared/state/workflow.service';
+import { WorkflowQuery } from 'app/shared/state/workflow.query';
+import { MyEntriesQuery } from 'app/shared/state/my-entries.query';
+import { SessionQuery } from 'app/shared/session/session.query';
+import { SessionService } from 'app/shared/session/session.service';
+import { MyWorkflowsService } from 'app/myworkflows/myworkflows.service';
+import { RegisterToolService } from 'app/container/register-tool/register-tool.service';
+import { Base } from 'app/shared/base';
+import { Dockstore } from 'app/shared/dockstore.model';
+
+@Component({
+  selector: 'app-entry-boxes',
+  templateUrl: './entry-boxes.component.html',
+  styleUrls: ['./entry-boxes.component.scss'],
+})
+export class EntryBoxesComponent extends Base implements OnInit {
+  Dockstore = Dockstore;
+  @Input() entryType: string;
```

Comment:
`@Input() entryType: EntryUpdateTime.EntryTypeEnum` would be more better.

Reply:
I'm currently just manually passing a string from the 'new-dashboard.component.html' into the 'entry-box' so it knows which one it is rendering

Reply:
Ugh, I can't believe I wrote `more better`. :( At any rate, I think you can still declare the type as `EntryUpdateTime.EntryTypeEnum`, and you can pass a string from the HTML, and the compiler would be able to verify that the string is a legit value (although we may have template validation turned off).

Reply:
haha oh nice I'll do that :+1: